core/src/apps/hedera/layout.py

```diff
@@ -0,0 +1,165 @@
+from trezor import ui
+from trezor.messages import ButtonRequestType
+from trezor.messages.HederaAccountAmount import accountID
+from trezor.messages import HederaSignTx
+from trezor.strings import format_amount
+from trezor.ui.components.tt.scroll import Paginated
+from trezor.ui.components.tt.text import Text
+
+from apps.common.confirm import require_confirm, require_hold_to_confirm
+
+
+def format_account(account: accountID) -> str:
+    shard = account.shard
+    realm = account.realm
+    num = account.num
+    return f"{shard}.{realm}.{num}"
+
+
+async def confirm_account(ctx, account_id: str):
+    text = Text("Confirm Account", ui.ICON_CONFIRM, ui.GREEN)
+    text.bold(account_id)
+    await require_confirm(ctx, text, ButtonRequestType.SignTx)
+
+
+async def confirm_create_account(ctx, initial_balance: str):
+    text = Text("Create Account", ui.ICON_SEND, ui.BLUE)
+    text.normal("Initial Balance:")
+    text.bold(initial_balance)
+    await require_hold_to_confirm(ctx, text, ButtonRequestType.SignTx)
+
+
+async def confirm_transfer_hbar(ctx, msg: HederaSignTx):
+    send_index = 0
+    receive_index = 1
+
+    if msg.tx.cryptoTransfer.transfers.accountAmounts[0].amount > 0:
```

Comment:
Missing check for `msg.tx.cryptoTransfer.transfers.accountAmounts[0].amount is not None`